test/test_vim.py

```diff
@@ -63,8 +64,9 @@ def test_strwidth():
 @with_setup(setup=cleanup)
 def test_chdir():
     pwd = vim.eval('getcwd()')
-    vim.chdir('/')
```

Comment:
Appears to be a problem in the core. I have an old version of neovim-qt (`0.1.3-dev`), and `:cd /` gives `E472: Command failed`. We're probably missing some Windows-specific handling somewhere.